### ISSUE TITLE
Unsuccessful "theme deploy"

### ISSUE THREAD

**Issue by sdomino** [Shopify/themekit#591]
**Describe the bug**
When trying to `theme deploy -a`, it appears to successfully upload all my theme files, however, when I visit the dashboard, it doesn't seem to be applying the changes (I can observe the changes when doing a `theme watch`).

<img width="844" alt="screen shot 2018-10-24 at 10 58 28 pm" src="https://user-images.githubusercontent.com/28181/47477286-b6d7a780-d7e1-11e8-8894-8c7cc5a7b9db.png">

I don't have any ignore configs in my `config.yml` and don't have an ignore file either. I've successfully done this many times in the past (using `theme upload`* or `theme replace`*), so I'm not sure what it's not working now. 

I'm not getting any type of error returned when running the command. When looking for changes in the Shopify dashboard the "Last Saved" dates for my themes are not updating, so this seems to indicate that nothing is actually being deployed (aside from the fact the changes aren't there either).

\* as a side note, I've also tried these commands, and aside from a deprication warning they also appear to "succeed", but not actually update the theme files

**To Reproduce**
1. Make changes to any file in the theme
2. run `theme deploy`
3. check for theme updates

**Expected behavior**
I would expect my theme files to be updated in Shopify after issuing a `theme deploy`

**Environment (please complete the following information):**
 - OS [e.g. iOS]: OSX 10.14
- Themekit version [e.g. v0.7.5]: ThemeKit 0.8.1 darwin/amd64
 - Editor [e.g. atom, sublime]: VSCode
 - Ignore: NONE

**Additional context**
I recently got a new laptop and since haven't been able to deploy. None of the configs/tokens have changed in my codebase, and I'm not getting any sort of error (authentication or otherwise).

**Comment by tanema** [comment]
Do the changes seem applied on multiple uploads? There is a known bug (but still undiagnosed) that file changes don't show up until the second save here: https://github.com/Shopify/themekit/issues/498

Also, I ask just run a `theme open` to make sure that you are editing the theme you think you are. I am sorry I know you probably have, I just have to eliminate the easy stuff. 

We can also try different versions of themekit

- **downgrade** `theme update --version=v0.7.5` 
- **prerelease** `theme update --version=v1.0.0-pre`

Lastly thank you for your thorough explanation, it really helps!

**Comment by sdomino** [comment]
I've tried to deploy probably 6ish times now with no luck (I do remember that behavior from past deploys, however).

In my `config.yml` I've got two nodes, "production" and "development". Both use the same `access_token`, `password`, and `store`. The only difference is `theme_id`.

When I run `theme open -e development/production` the `preview_theme_id` is the same ID that I have under my "development" and "production" nodes respectively (I haven't updated either ID since I originally configured `themekit` -- we only have the one store that I'm doing development on).

I've tried both the downgraded versions and prerelease version as suggested above and still no luck.

Is there a debug mode I could run in that provides more output than verbose? Everything appears to work when I deploy, just nothing is updated, and I'm not getting any errors or failed responses. Also, is there force update option (I noticed the help output doesn't indicate a `-f` or `--force`)?
